cmake-proxies/cmake-modules/AudacityTesting.cmake

```diff
@@ -115,6 +115,12 @@ if( ${_OPT}has_tests )
          # On macOS CMake will generate a placeholder that CTest fails to handle correctly,
          # so we have to setup the path manually
          set( audacity_target "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/$<CONFIG>/Audacity.app/Contents/MacOS/Audacity" )
+      elseif (WIN32)
+         set( audacity_target
+            powershell
+               -ExecutionPolicy Bypass
+               -File "${CMAKE_SOURCE_DIR}/tests/journals/test_runner.ps1"
+               "$<TARGET_FILE:Audacity>" )
```

Comment:
Maybe you will want to do all of this wrapping only conditionally and turn it off once you have debugged the present problems.

Reply:
I really want to have for every run. It doesn't produce to much output in successful runs anyway.

Reply:
As you wish